array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,17 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var newAr = [];
+  if (count < array.length) {
+    for (var i = 0; i < array.length - count; i++) {
+      newAr.push(array[i]);
+    }
+  } else {
+    return array;
```

Comment:
If the `count` is greater than the length of the array, you should return an empty array, since that will have "dropped" all entries in the array